cdds/cdds/workflows/conversion/flow.cylc

```diff
@@ -316,6 +316,17 @@ mkdir -p {{ BASE_OUTPUT }}
             --ntasks = 1
     {% endif %}
 
+    [[run_repack_{{ STREAM }}]]
+        inherit = {{ PLATFORM }}, COMPLETION_FAMILY_{{ STREAM }}
+        execution retry delays = PT1M, PT3M, PT5M
+        [[[environment]]]
+            ROSE_TASK_APP = repack_output
+            STREAM = {{ STREAM }}
+            PROC_DIR = {{ CDDS_CONVERT_PROC_DIR }}
```

Comment:
Can this be removed for simplicity? I think the logger can have the logging location set by default